examples/era5_observations_to_forcing_file.jl

```diff
@@ -0,0 +1,287 @@
+using NCDatasets
+using Statistics
+using Dates
+
+# for radiation calculation 
+using Insolation
+import Insolation.Parameters as IP
+import ClimaParams as CP
+
+time_resolution = 3600 # switch to 86400 for monthly data
+FT = Float64
+
+# parameters 
+R_d = 287.05
+g = -9.81
+rootdir = "/scratch/julian/ERA5/data_download/" # clima
+rootdir = "/Users/julianschmitt/Downloads/era5/" # local
+rootdir = "/central/groups/esm/jschmitt/era5/tv/july_forcing/" # hpc
+tvforcing = NCDataset(rootdir * "july2007_forcing_and_cloud_hourly_profiles.nc") # profile data
+tv_inst = NCDataset(rootdir * "july2007_hourly_inst.nc") # skt
+tv_accum = NCDataset(rootdir * "july2007_hourly_accum.nc") # slhf, sshf
+
+function get_horizontal_tendencies(lat, lon_index, lat_index, column_ds)
+    """
+    Computes horizontal tendencies for temperature and specific humidity at a given location.
+    """
+    rearth = 6378e3
+    lat_rad = deg2rad(lat)
+    coslat = cos(lat_rad)
+    dx = 2 * π * rearth * coslat / 360 * 0.25
+    dy = 2 * π * rearth / 360 * 0.25
+
+    # get velocities at site location
+    ᶜu = column_ds["u"][lon_index, lat_index, :, :]
+    ᶜv = column_ds["v"][lon_index, lat_index, :, :]
+
+    # get temperature at N S E W of center for gradient calculation
+    ʷT = column_ds["t"][lon_index - 1, lat_index, :, :]
+    ⁿT = column_ds["t"][lon_index, lat_index + 1, :, :]
+    ˢT = column_ds["t"][lon_index, lat_index - 1, :, :]
+    ᵉT = column_ds["t"][lon_index + 1, lat_index, :, :]
+
+    # get specific humidity at N S E W of center for gradient calculation
+    ʷq = column_ds["q"][lon_index - 1, lat_index, :, :]
+    ⁿq = column_ds["q"][lon_index, lat_index + 1, :, :]
+    ˢq = column_ds["q"][lon_index, lat_index - 1, :, :]
+    ᵉq = column_ds["q"][lon_index + 1, lat_index, :, :]
+
+    # temperature and specific humidity advective tendency at center 
+    tntha = -(ᶜu .* (ᵉT .- ʷT) ./ (2dx) .+ ᶜv .* (ⁿT .- ˢT) ./ (2dy))
+    tnhusha = -(ᶜu .* (ᵉq .- ʷq) ./ (2dx) .+ ᶜv .* (ⁿq .- ˢq) ./ (2dy))
+
+    return tntha, tnhusha
+end
+
+# compute vertical advection terms
+function get_vertical_tendencies(sim_forcing, var)
```

Comment:
it seems you only use this function to get the size of an array on L140, 142. Was this intentional? if so, you can safely remove the `get_vertical_tendencies` function and infer the size another way. 
You may also want to verify that setting `tntva` and `tnhusva` to 0 here is resulting in an eddy_vert_fluctuation tendency of 0 in `external_forcing_tendency!`